clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -118,11 +171,15 @@ func (c *clusterClient) GetClusterObjects() ([]*clusterv1.Cluster, error) {
 	return clusters, nil
 }
 
-func (c *clusterClient) GetMachineDeploymentObjects() ([]*clusterv1.MachineDeployment, error) {
+func (c *clusterClient) GetClusterObjects() ([]*clusterv1.Cluster, error) {
```

Comment:
Could you add comments to these preserved methods to indicate that they are deprecated and to be removed in the future? It might be good to also log the deprecation as well.

Reply:
Good suggestion. Added comments and log messages.